config.js

```diff
@@ -1,15 +1,26 @@
 import path from 'node:path';
-
 export const repoPath =
   process.env.MOZILLA_CENTRAL_REPO_PATH || '../mozilla-unified';
```

Comment:
Does setting `MOZILLA_CENTRAL_REPO_PATH` as an env var on windows also not work for you? 

It should be possible to add a `file://` prefixed path there to point to the location of your mozilla-central checkout.

Reply:
I've tried this locally and I'm not getting that to work though.